ui/src/cpu_inspector.rs

```diff
@@ -42,11 +49,24 @@ impl View for CpuInspector {
             }
             ui.text_edit_singleline(&mut cartridge_name);
             if ui.button("▶").clicked() {
-                // Start a thread for gameboy execution
-                todo!("Start a thread for a background gameboy execution");
+                if self.thread_handle.is_some() {
```

Comment:
It can be nice to make button "locked" for better understanding that is running :)